hrjobcontract/CRM/Hrjobcontract/Import/Form/SummaryBaseClass.php

```diff
@@ -104,5 +107,12 @@ public function preProcess() {
     foreach ($properties as $property) {
       $this->assign($property, $this->get($property));
     }
+
+    $isEntitlementUpdate = $this->_importMode == Import_Parser::UPDATE_ENTITLEMENTS;
+    $entitlementPageUrl = CRM_Utils_System::url(
+      'civicrm/admin/leaveandabsences/periods/manage_entitlements'
```

Comment:
Without passing any params, the system will calculate the entitlement for ALL contacts during the current period. This URL accepts a list of contacts to only do the calculation for them. Do you think it would be possible keep a list of the contacts whose contracts were imported and then pass their IDs to the URL?

Reply:
Did some investigations and it will be possible to do that but because of the data limitations of GET request, would leave as is, as agreed.